ui/components/report.jsx

```diff
@@ -30,6 +31,7 @@ var Report = React.createClass({
     return <div className="report">
       <Header passes={passes.length} failures={failures.length}
         duration={duration}/>
+      <Results data={this.props.data}/>
```

Comment:
`suites` would be a better name for the prop and you also use that name [here](https://github.com/uberVU/mocha-mugshot-reporter/blob/414b3f58fe60511c62dbd5bf08ea351bda2bfcc6/ui/components/report.jsx#L11).